src/styles/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
better have `box-sizing` moved to reset/normalize (or in your case index.scss) as most probably you plan to have same box-sizing for all your elements